source/onboarding/team/index.html.md.erb

```diff
@@ -36,12 +36,12 @@ Make sure you pick the Jenkins \[app] user:
 ## Github config
 
 ### Access
+Once you've received your `hmcts.net` email account and `Confluence` access, you can request GitHub access in one of the two ways.
 
-Request GitHub access through your PMO, (probably who you showed your documents to on your first day, 
-if in doubt ask your delivery manager).
+1. If you're the first person in your team you should raise a help request in #platops-help, asking for github access, you need to provide your GitHub username and say that you agree to [Acceptable Use Policy](https://tools.hmcts.net/confluence/display/RPE/Acceptable+Use+Policy+and+Contractor+Security+Guidance).
+2. If someone in your team already has access then ask them to follow the instructions at [github-management](https://github.com/hmcts/github-management) repository to add you.
 
-You should receive an email inviting you to the GitHub organisation, visit [https://github.com/hmcts](https://github.com/hmcts)
-to accept the invite.
+You should receive an email inviting you to the GitHub organisation, visit [https://github.com/hmcts](https://github.com/hmcts) to accept the invite.
```

Comment:
In the above lines I would probably swap 1 and 2 around, self service is preferred.